lib/plugins/cdn.js

```diff
@@ -23,6 +23,11 @@ const CDN_BUCKET = 'warehouse-cdn';
  */
 
 module.exports = fp(
+  /**
+   * Initialize CDN plugin
+   * @param {import('../warehouse').WarehouseApp} fastify
```

Comment:
Why is this type different than the `fastify` param in other plugins?

Reply:
https://github.com/godaddy/warehouse.ai/pull/112/files#diff-642dfcaadcf076d1d0bff936dcb0ac62ab947e6afa45e2356a1d31d2bb67cd8dR14

Reply:
It's an extension where I declare that there are also 2 properties call `dynamo` and `s3`.